code/__HELPERS/roundend.dm

```diff
@@ -203,14 +203,14 @@
 	//Print a list of antagonists to the server log
 	var/list/total_antagonists = list()
 	//Look into all mobs in world, dead or alive
-	for(var/datum/mind/Mind in minds)
-		var/temprole = Mind.special_role
-		if(temprole)							//if they are an antagonist of some sort.
-			if(temprole in total_antagonists)	//If the role exists already, add the name to it
-				total_antagonists[temprole] += ", [Mind.name]([Mind.key])"
-			else
-				total_antagonists.Add(temprole) //If the role doesnt exist in the list, create it and add the mob
-				total_antagonists[temprole] += ": [Mind.name]([Mind.key])"
+	for(var/datum/antagonist/A in GLOB.antagonists)
+		if(!A.owner)
+			continue
+		if(A.name in total_antagonists)	//If the role exists already, add the name to it
+			total_antagonists[A.name] += ", [key_name(A.owner)]"
+		else
+			total_antagonists.Add(A.name) //If the role doesnt exist in the list, create it and add the mob
```

Comment:
This line is completely unnecessary

This if/else could actually be shortened down to
`total_antagonists[A.name] += "[total_antagonists[A.name] ? "," : ":"] [key_name(A.owner)]"`
But that's a bit code golfy for what's reasonable.

Reply:
I'll just change this to a list and join it in output. I didn't really pay attention when changing the used name